### ISSUE TITLE
Convert ods to csv - possible bug !

### ISSUE THREAD

**Issue by ebocher** [miachm/SODS#16]
I try to read a sheet and convert it into a CSV file.

Code : 
``` java
        String inputFile = "See attached file";
        SpreadSheet spread = new SpreadSheet(new File(inputFile));
        System.out.println("Number of sheets: " + spread.getNumSheets());

        List<Sheet> sheets = spread.getSheets();

        for (Sheet sheet : sheets) {
            System.out.println("In sheet " + sheet.getName());

            Range data = sheet.getDataRange();

            Object[][] values = data.getValues();

            for (Object[] row : values) {
                ArrayList forJoin = new ArrayList();
                for (Object value : row) {
                    if (value instanceof Double) {
                        value = value.toString();
                    }
                    forJoin.add(value);
                }
                System.out.println(String.join(",", forJoin));
            }
        }
```

It returns the following result : 

```
Number of sheets: 1
In sheet fuller
id,lon,lat,nom_lieu,elephant_mer,baleine,cachalot,globicephal_noir,description
1.0,-72.063440,41.286780,New london,null,Départ le 15 juillet 1859.,null,null,null
```

instead of 

```
Number of sheets: 1
In sheet fuller
id,lon,lat,nom_lieu,elephant_mer,baleine,cachalot,globicephal_noir,description
1.0,-72.063440,41.286780,New london,null,null,null,null,Départ le 15 juillet 1859.



```

There is an offset that seems to be related to the null values. Or maybe I'm not using the library properly.

Thanks.

[example_ods.ods.zip](https://github.com/miachm/SODS/files/4224147/example_ods.ods.zip)

**Comment by ebocher** [comment]
Note that the offset appears when two null values follow each other

This ODS file is well parsed
```

id | cachalot | globicephal_noir | description |  
1 | non |   | Départ le 15 juillet 1859. |  


```
The next one no

```

id | cachalot | globicephal_noir | description |  
1 |  |   | Départ le 15 juillet 1859. |  


```

**Comment by miachm** [comment]
Thanks for your report!

It looks like a problem with repeated cells:

`<table:table-cell table:number-columns-repeated="4"/>`

ODS format uses the attribute "repeated" to save space in the file. The library takes that in consideration but deliberately ignores null cell values for optimization. Not a good idea.

It should be easy to fix, just taking an if away :)

**Comment by ebocher** [comment]
Thanks for your answer!
I'm not familiar with your lib. Can you point me to the resource to modify?

**Comment by miachm** [comment]
Don't worry about it! I can commit and release a new version when I have
the time.

Nevertheless, it's in the file OdsReader.java. If you search for the
variable last_cell_value.

**Comment by ebocher** [comment]
Try to play with last_cell_value variable but without  success.
So interested to get a fix as soon as you can.
Best regards

**Comment by miachm** [comment]
Solved in the v1.2.2

**Comment by ebocher** [comment]
Excellent thanks a lot.
